python/pyiceberg/io/__init__.py

```diff
@@ -55,6 +55,17 @@
 HDFS_PORT = "hdfs.port"
 HDFS_USER = "hdfs.user"
 HDFS_KERB_TICKET = "hdfs.kerberos_ticket"
+GCS_TOKEN = "gcs.oauth2.token"
+GCS_TOKEN_EXPIRES_AT = "gcs.oauth2.token-expires-at"
+GCS_PROJECT_ID = "gcs.project-id"
+GCS_ACCESS = "gcs.access"
+GCS_CONSISTENCY = "gcs.consistency"
+GCS_CACHE_TIMEOUT = "gcs.cache-timeout"
+GCS_REQUESTER_PAYS = "gcs.requester-pays"
+GCS_SESSION_KWARGS = "gcs.session-kwargs"
+GCS_ENDPOINT_URL = "gcs.endpoint-url"
```

Comment:
In other places, we have used just "endpoint" rather than "endpoint-url" to point the client to a different base URI. Should we do that here for consistency? Other examples are `glue.endpoint` and `s3.endpoint`.

Reply:
Great catch, I like that very much